public/static/locales/de/tenants.json

```diff
@@ -27,9 +27,5 @@
   "lacoqueta": {
     "title": "Bäume pflanzen mit La Coqueta",
     "description": "Zu sehen, wie ihre eigenen Kinder aufwachsen und sich zunehmend für die Klimakrise interessieren, hat Celia dazu inspiriert, ihre Marke als Plattform für positiven Wandel zu nutzen. Nachdem sie erfahren hatte, dass Bäume das wirtschaftlichste und effektivste Mittel sind, um CO2 zu binden und so mehr Zeit zu gewinnen, um die Treibhausgasemissionen auf Null zu reduzieren und die Klimakrise zu entschärfen, entschied sie sich, diese spannende Reise mit Plant-for-the-Planet anzutreten. Auch Sie können Teil dieser Reise werden und einige Bäume spenden - in Granada, der Heimatstadt von Celia, oder in anderen Gebieten auf der ganzen Welt."
-  },
-  "ulmpflanzt": {
-    "title": "Ulm pflanzt Bäume",
-    "description": "124.781 Bäume – für jede/n Ulmer Bürger/in einen. Das ist das Ziel von „Ulm pflanzt Bäume“ – der Klima-Initiative von Plant-for-the-Planet , dem unw (Ulmer Initiativkreis nachhaltige Wirtschaftsentwicklung e. V.), den Ulmer Unternehmen pervormance international, Trivis und der Volksbank Ulm sowie der Stadt Ulm und der lokalen agenda ulm 21. Pflanzen auch Sie mit und helfen Sie, das größte Aufforstungsprojekt der Menschheitsgeschichte zu realisieren. Für Sie – und für uns alle!"
```

Comment:
These have to be added in the English version as it is the default language and no resource can exists which does is not included in the English version!

Reply:
added